selfdrive/car/toyota/fingerprints.py

```diff
@@ -835,7 +837,7 @@
       b'8965B47022\x00\x00\x00\x00\x00\x00',
       b'8965B47023\x00\x00\x00\x00\x00\x00',
       b'8965B47050\x00\x00\x00\x00\x00\x00',
-      b'8965B47060\x00\x00\x00\x00\x00\x00',  # This is the EPS with good angle sensor
```

Comment:
same with these

Reply:
We can expand on these types of comments like I did with Honda and move to the top? Giving them more information is a good change in itself